glean-core/src/internal_pings.rs

```diff
@@ -0,0 +1,31 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use super::metrics::PingType;
+
+/// Glean-provided pings, all enabled by default.
+///
+/// These pings are defined in `glean-core/pings.yaml` and for now manually translated into Rust
```

Comment:
This will go away once we have the nice Rust API, right? If so, any chance you could add that as a comment here, mentioning the nice rust API bug number?

Reply:
Depends. It could definitely be auto-generated, though it's core internal so having that generated on top of the nice API could be tricky

Reply:
Right! It would be so cool , though :-P